src/components/App/App.jsx

```diff
@@ -22,17 +22,15 @@ function App() {
 	return (
 		<div>
 			{<Sidebar pageWrapId={"page-wrap"} outerContainerId={"outer-container"} />}
-			<div className="page-wrap">
```

Comment:
This should probably be a seperate PR/commit...

And you should remove the `pageWrapId` and `outerContainerId` props from the `Sidebar` component above. I verified that we don't need these since the [react-burger-menu](https://github.com/negomi/react-burger-menu) library doesn't require them for the slide animation we are using.